src/test/java/tt/TestZip.java

```diff
@@ -30,19 +25,6 @@ public static void main(String[] args) throws Exception
 
     
     
-    private static void testExport() throws Exception
```

Comment:
@alexdunnjpl @al-niessner @tloubrieu-jpl Just wanted to confirm we no longer need these tests here?

Reply:
can't give an informed answer, sorry